### ISSUE TITLE
Cannot debug with chrome adapter missing chromeDebug.js

### ISSUE THREAD

**Issue by stefhol** [daveleroy/sublime_debugger#148]
Sublime is a fresh install
tested on Windows 11 and Macos

```
⟸ process/starting :: ['/opt/homebrew/bin/node', '/Users/user/Library/Application Support/Sublime Text 3/Packages/Debugger/data/adapters/chrome/extension/out/src/chromeDebug.js']
⟸ process/started ::
⟸ request/initialize(1) :: {'clientID': 'sublime', 'clientName': 'Sublime Text', 'adapterID': 'chrome', 'pathFormat': 'path', 'linesStartAt1': True, 'columnsStartAt1': True, 'supportsVariableType': True, 'supportsVariablePaging': False, 'supportsRunInTerminalRequest': True, 'supportsMemoryReferences': True, 'locale': 'en-us'}
⟸ process/stderr :: node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/Users/user/Library/Application Support/Sublime Text 3/Packages/Debugger/data/adapters/chrome/extension/out/src/chromeDebug.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v17.3.0
```
<img width="678" alt="image" src="https://user-images.githubusercontent.com/35531567/152062131-a2a83c80-ecbf-49bf-9104-559881e33552.png">

**Comment by daveleroy** [comment]
They removed the old debugger but getting the new one working is going to be a lot of work

**Comment by daveleroy** [comment]
Resolved in 0.6.4
